_sass/elements/_links.scss

```diff
@@ -7,41 +7,76 @@
 // 
 // Styleguide elements.links
 
-a,
-.link-alpha {
-  border: none;
-  color: $base-font-color;
+%link {
   cursor: pointer;
   text-decoration: none;
+  border: none;
   transition: color 0.1s linear;
+}
+
+
+p a,
+.link-alpha {
+  @extend %link;
+  color: $blue;
+  text-decoration: underline;
+
   &:active,
   &:hover {
     text-decoration: none;
-    border-bottom: 1px dotted $base-link-color;
-    img{
-      border: none;
+
+    img, 
+    span {
+      text-decoration: none;
     }
   }
 
   &.button, 
   &.button-primary,
   &.button-secondary {
-    border: none;
+    text-decoration: none;
   }
-  img {
+  
+  img, 
+  span {
     border: none;
   }
 
 }
+
 .link-beta {
-  @extend a;
+  @extend %link;
   color: $white;
+}
+
+a,
+.link-charlie {
+  @extend %link;
+  color: $blue;
+  text-decoration: none;
   &:active,
   &:hover {
-    text-decoration: none;
-    border-bottom: 1px dotted $white;
-    img{
-      border: none;
-    }
+    text-decoration: underline;
+  }
+}
+
+.link-active,
+.link-active-alpha {
+  @extend .link-alpha;
+  text-decoration: underline;
+}
+
+.link-active-beta {
+  @extend .link-beta;
+  text-decoration: underline;
+}
+
+a.link-no_under {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `link-no_under` should be written in lowercase with hyphens